mowgli_etl/pipeline/wdc/wdc_offers_corpus.py

```diff
@@ -19,7 +20,8 @@ def __init__(self, *, wdc_json_file_path: Path):
         self.__file_length = 0
         with open(self.__file_path) as data:
             for line in data:
-                self.__file_length += 1
+                if detect(line) == "en":
```

Comment:
Rather than having two detect calls (here and line 35) you'd usually have a helper like `__is_valid_line(self, line)` which encapsulates this logic (and allows extending it in a single place, later).